app/controllers/indicadores_controller.rb

```diff
@@ -0,0 +1,8 @@
+class IndicadoresController < ApplicationController
+  VENTANA_DE_TIEMPO_EN_MESES = 3
```

Comment:
Capaz lo podríamos mover a una variable de ambiente y sino hacemos 3 por default.
```suggestion
  VENTANA_DE_TIEMPO_EN_MESES = ENV['VENTANA_DE_TIEMPO_EN_MESES'].to_i || 3
```

Reply:
La idea es que ese 3 venga como parametro, lo hardcodié para empezar a probar